samples/fan_in_fan_out/E2_GetFileList/__init__.py

```diff
@@ -1,14 +1,18 @@
 import os
+from os.path import dirname
 from typing import List
 
 def main(rootDirectory: str) -> List[str]:
 
     all_file_paths = []
-    # We walk the file system, looking for all files
-    # below "rootDirectory"
+    # We walk the file system
     for path, _, files in os.walk(rootDirectory):
-        # For each file, we add their full-path to the list
-        for name in files:
-            file_path = os.path.join(path, name)
-            all_file_paths.append(file_path)
+        # We copy the code for activities and orchestrators
+        if "E2_" in path:
```

Comment:
Is there a reason we only want the "E2_" files now?

Reply:
In my experimentation trying to adapt this tutorial to Python, it's pretty easy to accidentally pick up on the python virtualenviroment files when trying to back-up the file. When doing this, you'll end up with a sample that runs for ~20 minutes and backs up over 60k individual files or so.

I decided to go for the files inside the `E2_` directories because it was a simple way to avoiding that rabbit hole :)